hasjob/views/listing.py

```diff
@@ -162,13 +160,31 @@ def jobdetail(domain, hashid):
 
     is_bgroup = getbool(request.args.get('b'))
     headline = post.headlineb if is_bgroup and post.headlineb else post.headline
+    is_post_admin = is_siteadmin or post.admin_is(g.user) or (g.user and g.user.flags.is_employer_month)
```

Comment:
No, `JobPost` doesn't know about lastuser privileges. We should be using the permissions system for this – both the model and lastuser grant permission tokens to the current user, and the view requires a specific token.